### ISSUE TITLE
Long press popup for links

### ISSUE THREAD

**Issue by rossburton** [metabolist/metatext#181]
In many apps if you long press a link you get a popup menu of some sort: gmail and Twitter show a preview with Open/Copy/Share buttons, whereas Slack opens the Share sheet.  Both let me interact with the URL without opening it first.

Please implement one of these. I’m guessing the share sheet is easier and I’m absolutely fine with that (as personally I do this most to share links to Pocket).

**Comment by jgrg** [comment]
Yes, I was going to ask for the same thing. The action I want to perform most often is to copy the URL to the clipboard.

**Comment by NeuroWinter** [comment]
100% I am often copying links to paste into urlscan.io before opening them. With the amount of new people the number of spam links are growing as well!

**Comment by mirabilos** [comment]
Yesplease! Having no way to copy links to load them in a private browser window is… I’d even call it an insulting deficiency. This is a bug, not an enhancement.
